website/docs: stages cleanup (#21558)
goauthentik/authentik: website/docs/add-secure-apps/flows-stages/stages/user_delete/index.md

---
title: User delete stage
---

The User Delete stage deletes the current `pending_user`.

## Overview

> This stage is intended for unenrollment or account-removal flows where the user has already been verified and the flow should remove the account.

Reviewer: ```suggestion
This stage is intended for unenrollment or account-removal flows where an user account should be removed from authentik.
```

Author: 👍

## Configuration options

This stage has no stage-specific configuration options.

## Flow integration

> Use this stage near the end of an unenrollment flow after any confirmation, re-authentication, or policy checks have already happened.

Reviewer: ```suggestion
Use this stage near the end of an unenrollment flow after any confirmation, re-authentication, or policy checks have occurred.
```
